File: C#/Loon2MonoGame/LoonMonoGame-Lib/loon/utils/Bundle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace loon.utils
{
    public interface Bundle<T> : IArray
    {
	
        void Put(string key, T value);

        T Get(string key);

        T Get(string key, T defaultValue);

        T Remove(string key);

        T Remove(string key, T defaultValue);
        }
    }

File: Java/Loon-Neo/src/loon/utils/Bundle.java
package loon.utils;

/**
 * 全局通用存储类Bundle的接口
 * 
 * @param <T>
 */
public interface Bundle<T> extends IArray {

	void put(String key, T value);

	T get(String key);

	T get(String key, T defaultValue);

	T remove(String key);

	T remove(String key, T defaultValue);
}